component/authenticator/src/test/java/org/wso2/carbon/extension/identity/authenticator/passwordpolicy/test/PasswordChangeHandlerTest.java

```diff
@@ -362,70 +354,24 @@ public void testHandleEventForPasswordNonExpiredUserWithNonIdentityClaim()
         when(realmService.getTenantUserRealm(TENANT_ID)).thenReturn(userRealm);
         when(userRealm.getClaimManager()).thenReturn(claimManager);
         when(claimManager.getClaim(PasswordPolicyConstants.LAST_CREDENTIAL_UPDATE_TIMESTAMP_CLAIM_NON_IDENTITY)).thenReturn(claim);
+        when(org.wso2.carbon.identity.password.expiry.util.PasswordPolicyUtils.isPasswordExpired(
+                anyString(), anyString())).thenReturn(false);
 
-        String time = String.valueOf(System.currentTimeMillis());
-        Map<String, String> claimValueMap = new HashMap<>();
-
-        claimValueMap.put(PasswordPolicyConstants.LAST_CREDENTIAL_UPDATE_TIMESTAMP_CLAIM,null);
-        claimValueMap.put(PasswordPolicyConstants.LAST_CREDENTIAL_UPDATE_TIMESTAMP_CLAIM_NON_IDENTITY, time);
```

Comment:
why we removed these tests?

Reply:
Previously we use the methods to check the password expiry in this component itself. But now we are using the governance util method and this test is not reuqired.